boards/fysetc-cheetah-v2/board-definition.json

```diff
@@ -0,0 +1,11 @@
+{
+	"$schema": "../board-definition.schema.json",
+	"name": "Cheetah v2",
+	"manufacturer": "Fysetc",
+	"serialPath": "/dev/fysetc-cheetah-v2",
+	"firmwareBinaryName": "firmware-fysetc-cheetah-v2.bin",
+	"compileScript": "compile.sh",
+	"flashScript": "flash.sh",
+	"flashInstructions": "Flashing dfu-util -R -a 0 -s 0x08008000:leave -D klipper.bin",
```

Comment:
This is the information shown to the user in the RatOS configurator mcu interface, it should only be used if there's anything the user should be aware of for automatic flashing to work. For example, on the octopus they're required to remove the SD Card. You can leave it out if there's nothing to do.

Reply:
I have removed the flashInstructions string, I wasn't sure about if it was needed or not.